frameworks/AutoGluon/setup.sh

```diff
@@ -1,13 +1,23 @@
 #!/usr/bin/env bash
 HERE=$(dirname "$0")
+AMLB_DIR="$1"
+VERSION=${2:-"latest"}
+if [[ "$VERSION" == "latest" ]]; then
+    VERSION="master"
+fi
 
 # creating local venv
-. $HERE/../shared/setup.sh $HERE
+. ${HERE}/../shared/setup.sh ${HERE}
 #if [[ -x "$(command -v apt-get)" ]]; then
 #    SUDO apt-get install -y libomp-dev
 if [[ -x "$(command -v brew)" ]]; then
     brew install libomp
 fi
 
-cat $HERE/requirements.txt | sed '/^$/d' | while read -r i; do PIP install "$i"; done
-#PIP install --no-cache-dir -r $HERE/requirements.txt
+cat ${HERE}/requirements.txt | sed '/^$/d' | while read -r i; do PIP install "$i"; done
+
+if [[ "$VERSION" =~ ^[0-9] ]]; then
+    PIP install --no-cache-dir -U autogluon==${VERSION}
+else
+    PIP install --no-cache-dir -U -e git+https://github.com/awslabs/autogluon.git@${VERSION}#egg=autogluon
```

Comment:
notes: 
1. `-U` can be removed when specifying version number or when installing from git.
2. pip clones the entire repo when only the required branch would be necessary.
it could be more efficient to use sth like:
```
git clone --depth 1 --single-branch --branch ${VERSION} --recurse-submodules ${repo} ${HERE}/lib/${egg}
PIP install -e ${HERE}/lib/${egg}
```

```